File: src/neveroddoreven/myth/primo.py
primos = []

def isPrimo(agen) -> bool:

  if (agen / 2 == 1):
    return True

  if (agen % 2 == 0):
    return False

  for i in range(3, int(agen/2), 2):
    if (agen % i == 0):
      return False

  return True

for i in range(2, 422):

  if isPrimo(i):
    primos.append(i)

for primo in primos:

  print(f"For primo: {primo}")

  iD = primos.index(primo) - 1

  while iD >= 0:

    lesPrimo = primos[iD]

    div = primo / lesPrimo

    print(f"{div}", end="")
    iD = iD -1

  print("...")

File: src/JScript/primo.js
const readline = require('readline').createInterface({

	input: process.stdin,
	output: process.stdout

});

function isPrimo(agen) {

	if (agen / 2 == 1) {
		return true;
	}

	if (agen % 2 == 0) {
		return false;
	}

	for (var i=3; i<(agen/2); i+=2) {
		if (agen % i == 0) {
			return false;
		}
	}

	return true;
}

readline.question('Enter the range start: ', rangeStart => {

	rangeStart = parseInt(rangeStart);

	readline.question('Enter the range end: ', rangeEnd => {

		rangeEnd = parseInt(rangeEnd);

		for (var i=rangeStart; i<rangeEnd; i++) {

			console.log(`Number ${i} is primo = ${isPrimo(i)}`);

		}

		process.exit();

	});

});
